roles/third_party/ldap-install/tasks/configure_ssl.yml

```diff
@@ -24,14 +24,6 @@
   when:
     - not ldap_ssl_already_created.stat.exists
 
-- name:                     Configure SSL
```

Comment:
This is actually a feature (or better to say workaround). First step will always fail, and each and every one would fail after that if you don't reverse the order. What it does is:

Tries to create it with https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/roles/third_party/ldap-install/templates/certs.ldif.j2, fails always (whatever is in there), then renders the file with a same name but with a different order and **then ** it works (see https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/roles/third_party/ldap-install/templates/certsrev.ldif.j2 which is same as https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/roles/third_party/ldap-install/templates/certs.ldif.j2 but just with reverse order). I read the internet I think about this one, and couldn't find any option except this to not have it failing or manually intervening. Btw. just retested it now the way proposed here, and fails on initial install.

Reply:
The workaround is only needed, because the template certs.ldif.j2 has the wrong order. When you switch the two types it works. https://askubuntu.com/questions/936382/openldap-error-configuring-starttls-ldap-modify-other-e-g-implementation-sp